File: tests/manual/cross_lang_bench/lua/fib.lua
local function fib(n)
    if n < 2 then return n end
    return fib(n - 1) + fib(n - 2)
end

local start = os.clock()
for i = 1, 5 do
    print(fib(28))
end
print("elapsed:", os.clock() - start)

File: tests/manual/cross_lang_bench/python/fib.py
import time

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

start = time.perf_counter()
for i in range(1, 6):
    print(fib(28))
print("elapsed:", time.perf_counter() - start)
